design/design.md

```diff
@@ -0,0 +1,414 @@
+# ALBA implementation design document
+
+[@tolikzinovyev](https://github.com/tolikzinovyev) November 2024
+
+## Introduction
+
+This is the first design document for our Rust implementation of
+Approximate Lower Bound Arguments (ALBA) intended to guide the development
+of the library.
+We aim to create a high quality reference implementation of ALBA sufficient
+for accurately prototyping larger protocols utilizing this primitive.
+Applications that could benefit, at least in the Cardano universe, include
```

Comment:
Add also general examples for users outside of Cardano

Reply:
I mentioned voting protocols and multisignatures.